src/tests/Common/CLRTest.Jit.targets

```diff
@@ -370,6 +381,64 @@ for inputAssemblyName in glob.glob("*.dll"):
           proc.kill()
           sys.exit(1)
 
+      test_det = proc.returncode == 0
+
+      # Test determinism
+      if test_det:
+
+        hash = hash_file(inputAssemblyName)
+
+        ilasm_args = f'{os.environ["CORE_ROOT"]}/ilasm -output={inputAssemblyName} {ilasmSwitches} {disassemblyName}'
+        print(ilasm_args)
+        proc = subprocess.Popen(ilasm_args, shell=True)
+
+        try:
+            proc.communicate()
+        except:
+            proc.kill()
+            sys.exit(1)
+
+        if hash != hash_file(inputAssemblyName):
```

Comment:
Would it be better to compare the content of the whole file instead of just a hash?

Reply:
That seems simpler; I'll move this over, too.